spec/models/converters/app_converter_spec.rb

```diff
@@ -16,4 +16,17 @@
     end
   end
 
+  context '#to_compose_yaml' do
+    fixtures :apps, :services
+
+    it 'creates a yaml string from the given App' do
+      expect(subject.to_compose_yaml).to be_a String
+    end
+
+    it "creates a yaml representation with the same services as the App" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.